transfers/file-transfers.md

```diff
@@ -68,19 +68,38 @@ Or
   * /objects
 
 ## Transfer files from media object
-
-Files that have been updated by the donor within the past 30
+  Files that have been updated by the donor within the past 30
  days should be quarantined for 30 days to ensure that
  all virus definitions are up to date.
 
-* Use a write-blocker to connect the drive to the computer. See [Using Tableau Write Blockers](/digarch/transfers/using-tableaus.html){:target="_blank"} for instructions.  
-* Run [ft.sh](../software#ftsh){:target="_blank"} to create a transfer package.
+* Use a write-blocker to connect the drive to the computer. See [Using Tableau Write Blockers](/digarch/transfers/using-tableaus.html){:target="_blank"} for instructions.
+
+* Use [rsync](../software#rsync){:target="_blank"} to create a transfer package.
+
+### Rsync
+  Rsync is a command utility used for copying and syncing file locations, both locally and remotely. More information about rsync can be found on the [rsync documentation page](https://linux.die.net/man/1/rsync) and [installation instructions](../software#rsync) can be found in our software section.
+
+
 
 On Windows:
 
-* Start Cygwin from the desktop. A terminal like screen should appear.
+* Start WSL. A terminal like screen should appear.
 
-![](media/image2.png)
+On Mac:
+
+* Open Terminal.
+
+On all operating systems:
+
+* Enter ```rsync -arP targetpath destinationpath```
+  * Note: If using WSL first navigate to the host directory and select relevent drive for paths. An example path to the Ultrabay write blocker is:
+  ```/tmp/docker-desktop-root/mnt/host/d# ```
```

Comment:
Just want to be sure. I think these should be available as `/mtn/host/d` I'm not sure what the docker-desktop-root thing is, and it makes me worry that transfers might fail because of it.

Reply:
These are available via /mtn/host except for y:/DigArchDiskStation.